frontend/src/Components/AdminTools/CohortsTab.jsx

```diff
@@ -3,36 +3,40 @@ import axios from 'axios';
 
 
 export default function Cohorts(props) {
+    const { setFeedback } = props;
+
     const [ cohortsList, setCohortsList ] = useState([]);
     const [ cohortName, setCohortName ] = useState('');
     const [ tracker, setTracker ] = useState({});
+    const [ reload, setReload ] = useState(0);
 
-    const getCohortsList = async () => {
-        try {
-            const { data } = await axios.get('/api/cohorts');
-            setCohortsList(data.payload);
-            const map = {};
-            for (let elem of data.payload) {
-                map[elem.cohort_id] = elem.cohort;
+    useEffect(() => {
```

Comment:
i addresses the useEffect warnings by changing the async/await functions into chained promises that returned undefined. async/awaits return promises which useEffect doesnt really like. take a look and decide which way is better. im okay as long as the warnings are gone, everything is working, so that the console is not full of them

Reply:
I noticed you did that as well in your last PR (sorry I’m only on my Github phone app with limited options and it’s not that user friendly viewing the code). With the changes I did there is no more useEffect warnings. I’m fine with either approach as well